airgun/views/hostgroup.py

```diff
@@ -80,6 +80,10 @@ class ansible_roles(SatTab):
         TAB_NAME = 'Ansible Roles'
         resources = MultiSelect(id='ms-hostgroup_ansible_role_ids')
         pagination = PF4Pagination()
+        more_item = Text('//span[@class="pf-c-options-menu__toggle-button-icon"]')
+        select_pages = Text('//ul[@class="pf-c-options-menu__menu"]/li[6]/button')
+        available_role = '//div[@class="available-roles-container col-sm-6"]/div[2]/div'
+        assigned_role = '//div[@class="assigned-roles-container col-sm-6"]/div[2]/div'
```

Comment:
No need of this as this needs to be a part of the overridden multi-select widget

Reply:
I have removed all the previous locators and created a new widget called "MultiSelectNoFilter,"  and here I addressed the locators.